src/Symfony/Component/Messenger/Stamp/TransportConfigurationStamp.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+class TransportConfigurationStamp implements StampInterface
+{
+    private $delay = 0;
+
+    // TODO - add more stuff!
+    public function getDelay(): int
+    {
+        return $this->delay;
+    }
+
+    public function setDelay(int $delay): self
```

Comment:
We'd need a comment or some hint here to explain that the delay is in milliseconds (if it really is, I don't know).